plugins/modules/aci_epg.py

```diff
@@ -75,7 +75,7 @@
     type: str
   useg:
     description:
-    - Use C(yes) to create uSeg EPG and C(no) is used to create Application EPG.
+    - Use C(true) to create uSeg EPG and C(false) is used to create Application EPG.
     type: str
     choices: [ 'yes', 'no' ]
```

Comment:
should choices changed also?

Reply:
Also should we include mapping logic for these string "yes"/"no" choices consistently? Because if not mistaken apic expects the string.

Reply:
resolved